lib/index.js

```diff
@@ -2448,17 +2448,26 @@ exposeMethod('last');
  * _(filenames).through(jsonParser).map(function (obj) {
  *     // ...
  * });
+ *
+ * // All errors will be propagated as Highland errors
+ * _(['zz{"a": 1}']).through(jsonParser);
```

Comment:
Chain an `errors` here so that it's obvious the error is propagated and not thrown.

```javascript
_(['zz{"a": 1}']).through(jsonParser).errors(function (err) {
    console.log(err); // => SyntaxError: Unexpected token z
});
```